aboutlibraries-compose/gradle.properties

```diff
@@ -1,3 +1,5 @@
 POM_NAME=AboutLibraries Library
 POM_DESCRIPTION=AboutLibraries automatically detects all dependencies of a project and collects their information including the license. Optionally visualising it via the provided ui components.
-POM_ARTIFACT_ID=aboutlibraries-compose
\ No newline at end of file
+POM_ARTIFACT_ID=aboutlibraries-compose
+
+org.jetbrains.compose.experimental.jscanvas.enabled=true
```

Comment:
please revert this change as this is already part of the global gradle.properties